lightning-network-book.tex

```diff
@@ -40,7 +40,7 @@ \chapter{Introduction}
 We continue by explaining the theory and construction of the Lightning Network.
 We will understand how to construct payment channels via Revocable Sequence Maturity Contracts and how to solve the routing issues via Hashed Time Locked Contracts.
 These two forms of smart contracts allow for the Blockchain to be transformed from a transaction layer for payments to an enforcing layer of those contracts.
-Similar to the real world, most contracts will never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
+Similar to the real world, most contracts will never have to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
```

Comment:
haha sorry too late and @billygarrison has corrected this while merging anyway. I merged the other one so you already have your attribution. hope that is a fine process for you and thanks for your support!